layouts/_default/list.atom.xml

```diff
@@ -32,14 +32,211 @@
         {{- $copyright = $copyright | replace "&copy;" "©" -}}
         <rights>{{ $copyright | plainify }}</rights>
     {{- end }}
-    {{ with site.Author.name -}}
-        <author>
-            <name>{{ . }}</name>
-            {{ with site.Author.email }}
-                <email>{{ . }}</email>
-            {{ end -}}
-        </author>
-    {{- end }}
+    {{ $authors := "" }}
+    {{ if isset site "Author" }}
+        {{ $authors = site.Author }}
+    {{ else if isset site "Authors" }}
```

Comment:
This will never work; `site.Author` is a special singular case-sensitive variable (which is not used anywhere inside Hugo :P)

Ref: https://github.com/gohugoio/hugo/blob/2d766f13b983e1959c0863041247ca48e009b87f/hugolib/site.go#L543

Reply:
I am confused cause you're using it [here](https://github.com/kaushalmodi/hugo-atom-feed/blob/37b9e7ad911df2b8993ad3fc6e252f09b880c91d/layouts/_default/list.atom.xml#L35) in your own code.

Reply:
My "never work" comment is for 

> if isset site "Authors" 

I am using `site.Author..` in that referenced line (note the uppercase and singular Author).